### ISSUE TITLE
Add element.clndr.instance() method

### ISSUE THREAD

**Issue by SNiels** [kylestetz/CLNDR#207]
This method should return the instance of the initialized calendar on that element.

**Comment by mikegioia** [comment]
@SNiels there's an undocumented way of getting this by accessing `.data( 'plugin_clndr' )` on the HTML element that the CLNDR was instantiated on.

However, having .getInstance() or .instance() is a much cleaner method.

**Comment by mikegioia** [comment]
@SNiels I just submitted a proposed update for this in the "version-1.3" branch. Let me know if you have a chance to take a look at it to see if it's sufficient.

You can now call `$(selector).clndr()` without any arguments on an instantiated calendar to return back the CLNDR instance.

**Comment by mikegioia** [comment]
This is resolved in 1.3.0

**Comment by SNiels** [comment]
I'm using the undocumented way at the moment because calendarContainr.clndr() will instantiate a clndr without params if it's not already instantiated (which I don't know at that point of the code). I would go the .instance route.
```js
calendar = calenderContainer.data('plugin_clndr');
if (calendar) {
         calendar.destroy();
}
calendar = calenderContainer.clndr(options);
```
